templates/qa_srl/templates.yaml

```diff
@@ -0,0 +1,57 @@
+dataset: qa_srl
+templates:
+  3cb09519-032e-4e51-bb97-47aa18ab4367: !Template
+    id: 3cb09519-032e-4e51-bb97-47aa18ab4367
+    jinja: 'Use the following context and list of answers to generate a possible question:
+
+
+      Context: {{sentence}}
+
+
+      Answers: {{answers|join(", ")}} |||
+
+      {{question|join(" ")|replace("_","")}} '
+    name: context_answer
+    reference: ''
+  4c14e864-0e9f-4a66-8978-4150ae8fb3d8: !Template
+    id: 4c14e864-0e9f-4a66-8978-4150ae8fb3d8
+    jinja: 'Use the following list of answers to generate a possible context-question
+      pair:
+
+
+      {{answers|join(", ")}} |||
+
+      {{sentence}}
+
+      {{question|join(" ")|replace("_","")}} '
+    name: aq
+    reference: ''
+  7caf24bf-ebe2-40bc-9ab8-a8a95f7e9c0d: !Template
+    id: 7caf24bf-ebe2-40bc-9ab8-a8a95f7e9c0d
+    jinja: 'Use the following context to generate a question:
+
+
+      {{sentence}} |||
+
+      {{question|join(" ")|replace("_","")}} '
+    name: context_qn
+    reference: ''
+  bec2ce78-fb31-4529-8b13-240fa6c8bc88: !Template
+    id: bec2ce78-fb31-4529-8b13-240fa6c8bc88
+    jinja: 'Identify the predicate in this sentence:
```

Comment:
This template results in duplicate examples since some examples in the dataset differ only in the question. I'm not sure if we have a way to deal with this, I will ask around.

Reply:
Checked and we are going to do post-prompting deduping, so this looks fine.